certified-connectors/The Bot Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,329 @@
+				"description": "Setup a new attribute in your bot",
+				"operationId": "CreateUserAttribute",
+				"parameters": [{
+					"name": "body",
+					"in": "body",
+					"required": true,
+					"schema": {
+						"type": "object",
+						"properties": {
+							"data": {
+								"type": "object",
+								"properties": {
+									"type": {
+										"type": "string",
+										"description": "type",
+										"title": "",
+										"default": "userattribute",
+										"x-ms-visibility": "internal"
+									},
+									"attributes": {
+										"type": "object",
+										"properties": {
+											"name": {
+												"type": "string",
+												"description": "Provide a name for your attribute",
+												"title": "Attribute name"
+											},
+											"is_pii": {
+												"type": "string",
+												"description": "is it personally identifiable information?",
+												"title": "is pii?",
+												"enum": [
+													"Yes",
+													"No"
+												]
+											}
+										},
+										"description": "attributes"
+									}
+								},
+								"description": "data",
+								"required": [
+									"type"
+								]
+							}
+						},
+						"required": [
+							"data"
+						]
+					}
+				}]
+			}
+		},
+		"/v1.0/users/{emailaddress}": {
+			"patch": {
+				"responses": {
+					"204": {
+							"description": "No content"
+						},
+						"500":
+							{
+								"description": "Internal Server Error"
+							}
+				},
+				"summary": "Set an attribute value",
+				"description": "Set an attribute value via the users email address",
+				"operationId": "SetUserAttribute",
+				"parameters": [{
+						"name": "emailaddress",
+						"in": "path",
+						"required": true,
+						"type": "string",
+						"description": "email address",
+						"x-ms-summary": "email address of the user who's attribute is being set"
+					},
+					{
+						"name": "body",
+						"in": "body",
+						"required": true,
+						"schema": {
+							"type": "object",
+							"properties": {
+								"data": {
+									"type": "object",
+									"properties": {
+										"type": {
+											"type": "string",
+											"description": "type",
+											"title": "",
+											"enum": [
+												"user"
+											],
+											"x-ms-visibility": "internal",
+											"default": "user"
+										},
+										"attributes": {
+											"type": "object",
+											"properties": {
+												"state": {
+													"type": "array",
+													"items": {
+														"type": "object",
+														"properties": {
+															"userattribute": {
+																"type": "object",
+																"properties": {
+																	"id": {
+																		"type": "string",
+																		"description": "Existing attributes in the bot",
+																		"title": "Attribute ID",
+																		"x-ms-dynamic-values": {
+																			"operationId": "Pullattributes",
```

Comment:
@tbpsami The operationId "Pullattributes" could not be found in this swagger document. Maybe you meant to say "PullAttributes"?